packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -431,8 +430,8 @@ class DocumentCollection {
     indexedFields = indexedFields
       ? indexedFields
       : getIndexFields({ sort, selector })
-    const indexId = options.indexId || getIndexNameFromFields(indexedFields)
-
+    const indexName =
+      options.indexId || `_design/${getIndexNameFromFields(indexedFields)}`
```

Comment:
nit: est-ce que ça vaudrait le coup de mettre plutôt `_design` en variable quelque part plutôt qu'en dur ici ?

Reply:
Why not, mais on ne l'utilise qu'ici, du coup pas sur que ce soit utile